packages/shared/src/components/image/VideoImage.tsx

```diff
@@ -7,18 +7,21 @@ import { ImageProps, Image } from './Image';
 export interface VideoImageProps extends ImageProps {
   size?: IconSize;
   wrapperClassName?: string;
+  wrapperWidth?: 'w-full' | 'w-fit';
```

Comment:
Isn't that what the wrappedClassName should do?

Reply:
First thought about adding `!w-fit` to wrapperClassName, but then you get an extra classname.

But I will do some more testing and see if w-full is actually needed in production

Reply:
Cool yeah please check as I'm not sure for which place it was added as it's a generic component basically.

Reply:
The change from w-fit to w-full was in a different component, done in #2436.

If I set it to be w-fit always, the normal cards looks wrong
![image](https://github.com/dailydotdev/apps/assets/1681525/bf31431d-f50e-4f25-a395-a6a7af4462d0)

Reply:
Though, I think I actually can just add an empty div around the imagecomponent here to fix it.